docs/content/guide/halo/typescript.md

```diff
@@ -57,3 +54,53 @@ const client = new Client();
 :::
 
 Before data can be manipulated, a [`space`](../echo/typescript/README.md) must be obtained.
+
+## Receiving a device join invitation
+We often want to add a device to the same [HALO identity](../halo/).
+You can accept a device join invitation by providing the invitation code and the auth code to the dxos client.
+This example demonstrates how to accept a device join invitation in a console application.
+A device join invitation can be obtained from [the shell](../halo/#shell) or generated in typescript from an existing identity.
+
+```ts{7}
```

Comment:
make these snippets?

Reply:
Can do, although they don't seem to load and we duplicate the code across both the markdown and the snippet in the examples I've seen? I can look into it.

Reply:
see any of the quickstarts for the code-fence syntax and then run `pnpm nx apidoc docs` to propagate the snippets into the md files

Reply:
Ahh. `apidoc docs` is the secret sauce I was missing.